lib/params.js

```diff
@@ -51,11 +51,21 @@ function getIndentationFromStylelintRules (rules) {
   }
 }
 
-function loadConfig (stylelint, file, configFile) {
-  return stylelint.getConfigForFile(file, configFile).then(function (stylelintrc){
-    return stylelintrc && stylelintrc.config && stylelintrc.config.rules
+function loadConfig (stylelint, file, options) {
+  return Promise.resolve().then(function (){
+    if (file && !options.ignoreDisables) {
+      return stylelint.isPathIgnored(file)
+    }
+  }).then(function (isIgnored) {
+    if (isIgnored) {
+      return null
+    }
+    return stylelint.getConfigForFile(file, options.configFile).then(function (stylelintrc) {
+      return stylelintrc && stylelintrc.config && stylelintrc.config.rules
+    })
   }).catch(function (err) {
```

Comment:
i'd like to use `catch()` in the bottom of this method chain (after here https://github.com/morishitter/stylefmt/pull/231/files#diff-f5008a4d411fbb1dbee72fa9d191a862R78 )